lib/constants.ts

```diff
@@ -171,3 +171,17 @@ export const maxCachedBuckets = process.env.METADATA_MAX_CACHED_BUCKETS ?
     Number(process.env.METADATA_MAX_CACHED_BUCKETS) : 1000;
 
 export const validRestoreObjectTiers = new Set(['Expedited', 'Standard', 'Bulk']);
+
+export const bucketIndexes = {
```

Comment:
do we need to keep these constants here?
they seems more closely related to backbeat, can they not stay there, along with the rest of lifecycle code?